packages/terra-worklist-data-grid/tests/jest/Cell.test.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
+import Cell from '../../src/subcomponents/Cell';
+
+describe('Cell', () => {
+  it('verifies that only a row header cell(TH) is created when isRowHeader prop is true', () => {
```

Comment:
What is cell(TH)?

Reply:
I think you mean the `th` html tag in which case, can we update this to:
```suggestion
  it('verifies that only a row header cell (<th>) is created when isRowHeader prop is true', () => {
```